hoprd/rest-api/src/lib.rs

```diff
@@ -290,7 +284,6 @@ async fn build_api(
                 .route("/messages/peek", post(messages::peek))
                 .route("/messages/peek-all", post(messages::peek_all))
                 .route("/messages/size", get(messages::size))
-                .route("/messages/websocket", get(messages::websocket))
```

Comment:
This is not backwards-compatible.

Reply:
I will add the revertable single commit to return the functionality.